sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -197,7 +198,7 @@
  *
  * <pre>{@code
  * PCollection<AvroAutoGenClass> records =
- *     p.apply(AvroIO.read(...).from(...).withBeamSchemas(true);
+ *     p.apply(AvroIO.read(...).from(...).withBeamSchemas(true));
```

Comment:
:+1:

Reply:
very strange. On my repo [AvroIO](https://github.com/anantdamle/beam/blob/schema_support_cleanup/sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java) already has that extra closing parenthesis.
Is there something wrong I am doing?

Reply:
maybe someone updated it in the meantime. Do a `git pull origin master --rebase` from your branch just to be sure.

Reply:
sorry my mistake in reading the git messages. 
all ok.